docs/typescript.md

```diff
@@ -90,26 +104,26 @@ You can type the props of styled components.
 import styled from 'react-emotion'
 
 type ImageProps = {
-  src: string,
+  src: string;
   width: number;
 }
 
-const Image0 = styled('div')`
-  width: ${(props: ImageProps) => props.width};
+const Image0 = styled<ImageProps, 'div'>('div')`
+  width: ${(props) => props.width};
   background: url(${(props: ImageProps) => props.src}) center center;
```

Comment:
Should `props: ImageProps` be changed to `props` here?

Reply:
@mitchellhamilton Oh, thanks! I will change it.